core/standalone/build.gradle

```
@@ -140,6 +144,15 @@ processResources {
     }
 }
 
+// Ignores the property `git.build.time` in `git.properties` files to allow `compileScala` task to be cached
```

The task will be cached whether or not this property is ignored. Here's a better wording.
```suggestion
// Ignores the property `git.build.time` in `git.properties` files to allow `compileScala` task to be re-used from the cache when the timestamp changes.
```

That being said, the compileScala task should not use runtimeClasspath as an input. Is it doing so? Can you share the comparison?

---

I've been digging a little into the issue since I couldn't replicate it.
Apparently updating the `com.gorylenko.gradle-git-properties` to the `2.4.1` version caused the property `git.build.time` not to be included anymore into the `git.properties`, so the normalization is not needed anymore, I'll remove it.
Anyhow, just for the sake of understanding things, I tried rolling back the version of the plugin to the `2.0.0` and the task whose cache couldn't be reused was the `core:standalone:compileTestScala` and not the `compileScala` as I mistakenly said in the comment.
That said, I'll remove the normalization altogether.

---

Removed with 67447e90339eb2ff18a9092c2fd2051e23a0d310